src/alarms/alarm-handling.md

```diff
@@ -3,60 +3,69 @@
 That document specifies alarm handling for thin-edge.
 
 # Data Semantic of Alarms

```

Comment:
One key alarm data semantic that's followed in this spec is that an alarm of a given type and severity is different from another alarm of the same type with a different severity, i.e, a CRITICAL thermostat alarm and a MAJOR thermostat alarm are two different entities and **not related** at the data model level. The relation between the severity types of a given alarm type, if needed, must be maintained by the user. May be it's worth pointing that out in this section as his isn't a natural/human way of perceiving an alarm data.

The obvious side effect is that, when someone wants to detect the "latest" state of an alarm type, he'll have to read messages from 4 different severity topics and compare their timestamps to detect the most recent one. And similarly, once you've reacted to an alarm with some corrective action, you'll have to send 4 different messages to 4 different severity topics to clear them all.

Reply:
See comment above (https://github.com/thin-edge/thin-edge.io-specs/pull/18#discussion_r760835462).

Reply:
Yeah I had seen that comment. I was just suggesting that may be it's worth adding that significant assumption on the alarm data semantics to this section.